clang/include/clang/Basic/Diagnostic.h

```diff
@@ -946,6 +953,24 @@ class DiagnosticsEngine : public RefCountedBase<DiagnosticsEngine> {
     return (Level)Diags->getDiagnosticLevel(DiagID, Loc, *this);
   }
 
+  /// Diagnostic suppression mappings can be used to ignore diagnostics based on
+  /// the file they occur in. Mapping file is expected to be a special case list
```

Comment:
How about, instead "to ignore diagnostics based on the file they occur in", say "to avoid generating specific diagnostics in specific files" or perhaps "to suppress" or something similar?
I'm not sure "ignore diagnostics" clarifies the behavior.